server/submission/resolvers.js

```diff
@@ -48,9 +49,23 @@ async function setupCorrespondingAuthor(user, manuscript) {
     manuscript.manuscriptPersons.push(manuscriptPerson)
 
     if (!manuscriptPerson.user) {
+      const { email, firstName, lastName } = manuscript.submissionMeta.author
+      const { title, id } = manuscript
+
+      const emailToken = authentication.token.create({ username: email })
+
       await mailer.send({
-        to: manuscript.submissionMeta.author.email,
+        to: email,
         text: 'Please verify that you are a corresponding author',
+        html: `<p>${firstName} ${lastName} has recently submitted a paper to eLife, with the title "${title}". You have been listed as the corresponding author
+        so you will be contacted with any questions and an initial decision once the editors have completed their evaluation.</p>
+        
+        <button><a href="${config.get(
+          'pubsweet-server.baseUrl',
+        )}/confirm-author/${id}&token=${emailToken}">Confirm</a></button> <button><a href="${config.get(
+          'pubsweet-server.baseUrl',
+        )}/decline-author/${id}"
+        )}">Decline</a></button>`,
```

Comment:
Is it legal HTML to have a link inside a button?

Reply:
@jsms90 looks like @tamlyn is right:
```
The element “a” must not appear as a descendant of the “button” element.
From line 8, column 11 to line 8, column 22
Code Extract:

<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>title</title>
  </head>
  <body>
  <button><a href="#"></a></button>
  </body>
</html>
```

Reply:
Yeah sorry. Again, placeholder nonsense.

This should be a link that's styled to look like a button.